services/core-api/app/templates/email/mine_party_appt/emli_new_eor_email.html

```diff
@@ -0,0 +1,149 @@
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <style>
+        @media only screen and (max-width: 620px) {
+
+            table.body .wrapper,
+            s table.body .article {
+                padding: 10px !important;
+            }
+
+            table.body .content {
+                padding: 0 !important;
+            }
+
+            table.body .container {
+                padding: 0 !important;
+                width: 100% !important;
+            }
+
+            table.body .main {
+                border-left-width: 0 !important;
+                border-radius: 0 !important;
+                border-right-width: 0 !important;
+            }
+
+            table.body .btn table {
+                width: 100% !important;
+            }
+
+            table.body .btn a {
+                width: 100% !important;
+            }
+
+            table.body .img-responsive {
+                height: auto !important;
+                max-width: 100% !important;
+                width: auto !important;
+            }
+        }
+    </style>
+</head>
+
+<body
+    style="background-color: #ffffff; font-family: sans-serif; -webkit-font-smoothing: antialiased; font-size: 14px; line-height: 1.4; margin: 0; padding: 0; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%;">
```

Comment:
Future improvement (for the backlog): Looking into a templating language for these emails (handlebars?) seeing at the styling will be 90% the same. Would be nice if this was reduced to

```
<core-email>
    <paragraph>
     New Engineer of Record at {{ mine.mine_name }} requires Ministry
                                            acknowledgement.
     </paragraph>
     <paragraph>
            As of {{ start_date }}, a new Engineer of Record, {{ party.first_name }}
            {{ party.last_name }}, has been assigned
            to {{ tsf_name }} at {{ mine.mine_name }} and requires <a
                href="{{ core_appt_link }}">Ministry acknowledgement</a> to allow
            for
            the mine's compliance.

     </paragraph>

     <cta href="....">
          View Engineer of Record
     </cta>

    <footer>
</core-email>
```

Reply:
I agree. It's now 4979!